GroupMeClientApi/Models/Contact.cs

```diff
@@ -0,0 +1,153 @@
+        /// </summary>
+        [JsonProperty("created_at_iso8601")]
+        public string CreatedAtIso8601Time { get; internal set; }
+
+        /// <summary>
+        /// Gets the time when the <see cref="Contact"/> was created.
+        /// </summary>
+        public DateTime CreatedAtTime => DateTimeOffset.FromUnixTimeSeconds(this.CreatedAtUnixTime).ToLocalTime().DateTime;
+
+        /// <summary>
+        /// Gets a value indicating whether the <see cref="Contact"/> is hidden.
+        /// </summary>
+        [JsonProperty("hidden")]
+        public bool Hidden { get; internal set; }
+
+        /// <summary>
+        /// Gets the <see cref="Contact"/>'s unique identifier.
+        /// </summary>
+        [JsonProperty("id")]
+        public string Id { get; internal set; }
+
+        /// <summary>
+        /// Gets the MRI of a <see cref="Contact"/>. Unsure of what this value represents.
+        /// </summary>
+        [JsonProperty("mri")]
+        public string Mri { get; internal set; }
+
+        /// <summary>
+        /// Gets a <see cref="Contact"/>'s full name or username.
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name { get; internal set; }
+
+        /// <summary>
+        /// Gets the reason for a <see cref="Contact"/>. Unsure of what this value represents.
+        /// </summary>
+        [JsonProperty("reason")]
+        public int Reason { get; internal set; }
+
+        /// <summary>
+        /// Gets the Unix Timestamp when the <see cref="Contact"/> was last updated.
+        /// </summary>
+        [JsonProperty("updated_at")]
+        public int UpdatedAtUnixTime { get; internal set; }
+
+        /// <summary>
+        /// Gets the ISO 8601 timestamp when the <see cref="Contact"/> was last updated.
+        /// </summary>
+        [JsonProperty("updated_at_iso8601")]
+        public string UpdatedAtIso8601Time { get; internal set; }
+
+        /// <summary>
+        /// Gets the time when the <see cref="Contact"/> was last updated.
+        /// </summary>
+        public DateTime UpdatedAtTime => DateTimeOffset.FromUnixTimeSeconds(this.UpdatedAtUnixTime).ToLocalTime().DateTime;
+
+        /// <summary>
+        /// Gets the user id for a <see cref="Contact"/>.
+        /// </summary>
+        [JsonProperty("user_id")]
+        public string UserId { get; internal set; }
+
+        /// <summary>
+        /// Gets the <see cref="GroupMeClient"/> that manages.
+        /// </summary>
+        public GroupMeClient Client { get; internal set; }
+
+        /// <summary>
+        /// Blocks this <see cref="Contact"/>.
+        /// </summary>
+        /// <returns>True if this <see cref="Contact"/> is successfully blocked.</returns>
+        public async Task<bool> BlockContact()
+        {
+            this.Client.WhoAmI();
+            var request = this.Client.CreateRestRequest($"/blocks?user={this.Client.Me.Id}&otherUser={this.Id}", Method.POST);
```

Comment:
Client.Me is internal, but if we could avoid coupling here, that would be nice. WhoAmI returns a Member, and the recommended method for external API callers to consume this method is
```csharp
var me = this.Client.WhoAmI();
// use me object locally
```

This will keep this class isolated if the caching on Client.Me changes in the future, goes private, or is eliminated.